pages/product/[code].js

```diff
@@ -271,13 +269,23 @@ export default function DetailPage() {
                   alt={data.product.product_name}
                 />
                 <StyledCheck>
-                  {additivesFromStorage.length > 0 ? (
+                  {additivesFromServer.length > 0 ? (
                     <StyledPAdditives>
                       Additive:
                       {filteredAdditives.length > 0 ? (
-                        <SVGIcon variant="bad" width="20px" color="red" />
+                        <SVGIcon
+                          aria-label="bad"
```

Comment:
from just reading your code I wouldn't know what the aria-labels "bad" and "good" are supposed to tell me.

Reply:
first i thought to put the variant .. but in fact you are right to name them german i will rework this